refinery/templates/registration/registration_form.html

```diff
@@ -101,12 +101,19 @@ <h4>Terms of Service</h4>
 			<p>
         {{ form.spam_filter }}
       </p>
-
+      <script src='https://www.google.com/recaptcha/api.js'></script>
+      {% if recaptcha_error_message %}
+			  <p class="errors">
+          {{ recaptcha_error_message }}
+        </p>
+			{% endif %}
+      <div class="g-recaptcha" data-sitekey={{ REFINERY_GOOGLE_RECAPTCHA_SITE_KEY }}></div>
+      <br>
```

Comment:
Uneven indentation and formatting in this block.

Reply:
The formatting looks fine in PyCharm for me. I think this is just how GitHub is rendering it.

Reply:
This is because some of the whitespace chars are actually tabs (rendered in PyCharm as two spaces and in Github as eight). They should all be spaces.